### DISCUSSION TITLE
Does overseerr honor the delay between usenet and torrents?

### DISCUSSION THREAD

**Discussion by famewolf** [sct/overseerr#3417] [Q&A] [answered]
In sonarr and radarr I've set up searches so torrents are delayed by 1440 minutes while usenet search occur right away.  I did a search where it submitted the request to my torrent client within minutes while I personally was able to find the item in question via nzbhydra on usenet.

**Comment by TheCatLady** [comment, marked as answer]
Overseerr doesn't search for media or interact with torrent/usenet clients. Overseerr simply sends requests to the *arrs for processing.

**Comment by famewolf** [comment]
That's what I thought but based on results I figured I'd ask.  Thanks!  Apologies to anyone who got my deleted comment.  Replied to the wrong email!
